tools/tpm2_createek.c

```diff
@@ -399,15 +507,6 @@ static tool_rc tpm2_tool_onrun(ESYS_CONTEXT *ectx, tpm2_option_flags flags) {
         goto out;
     }
 
-    /* override the default attrs */
-    set_default_obj_attrs();
-
-    /* set the auth policy */
-    set_default_auth_policy();
-
-    /* set the default hierarchy */
-    set_default_hierarchy();
-
```

Comment:
@idesai went through some work items to make all the tools look more similar, I think this works against his end goal.

Reply:
This tool can take the patch as the changes haven't been made yet.

Reply:
`set_default_hierarchy()` could keep its old location because it doesn't depend on the `nameAlg` settings.